src/block_subs.h

```diff
@@ -87,6 +87,30 @@ struct BLOCK_CTRL {
 	} W;
 };
 
+GMT_LOCAL struct GMT_KEYWORD_DICTIONARY module_kw[] = { /* Local options for all the block* modules */
+	/* separator, short-option, long-option, short-directives, long-directives, short-modifiers, long-modifiers */
+	{ 0, 'A', "fields", "", "", "", "" },
+	{ 0, 'C', "center", "", "", "", "" },
+#if defined(BLOCKMODE)	/* Only blockmode has a -D option */
+	{ 0, 'D', "bin-width", "", "", "a,c,h,l", "average,center,high,low" },
+#endif
+#if defined(BLOCKMEDIAN)	/* Only blockmedian has a -T option */
```

Comment:
The comment is wrong:
```
/* Only blockmedian has a -T option */
```

Reply:
No, it is correct I think.  Of the three blockm*.c programs, only blockmedian has a -T option.

Reply:
Oh I see what you mean, this is -E... will fix